### PR TITLE
Added Pagination

### PR DESCRIPTION
**Pull request by rmaheshkumarblr**
Added Pagination for the posts in the Home page.  Based on https://jekyllrb.com/docs/pagination/

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -25,3 +25,8 @@ collections:
     output: true
     permalink: /notebooks/:path/
 
+
+# Gems
+gems: [jekyll-paginate]
+paginate: 2
```
Comment (EntilZha) [inline _config.yml]: Lets kick this up above 2, with the current layout 4 or 5 seems fine.

File: index.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
 </div>
 <h3 class="page-heading">Recent News</h3>
 <ul class="post-list">
-{% for post in site.posts %}
+{% for post in paginator.posts limit:4 %}
```
Comment (EntilZha) [inline index.html:17]: Make this match whatever is above for the pagination limit

File: index.html [hunk 3]
```diff
@@ -27,3 +27,16 @@ <h3 class="panel-title"><a href="{{ post.url | prepend: site.baseurl }}">{{ post
     </div>
 {% endfor %}
 </ul>
+
+<div class="pagination"> {% if paginator.previous_page %}
+      {% if paginator.previous_page == 1 %}
+      <p class="prevLink"><a href="{{site.baseurl}}/" class="previous">newer entries</a></p>
```
Comment (EntilZha) [inline index.html]: Perhaps "Newer Posts" is better. Is there any way to provide numeric pagination?

**Review by EntilZha — CHANGES_REQUESTED** [review]
Overall, solid improvement. Mostly just need to change the pagination from 2 to something higher since that seems a bit low

File: _config.yml (hunk 1 above)
Comment (rmaheshkumarblr) [reply _config.yml]: Updated to 4

File: index.html (hunk 2 above)
Comment (rmaheshkumarblr) [reply index.html:17]: Maintaining it at 4 as it is updated in the _config file

File: index.html (hunk 3 above)
Comment (rmaheshkumarblr) [reply index.html]: image

Based on https://jekyllrb.com/docs/pagination/

**Comment by EntilZha** [comment]
If there are no further changes LGTM

**Review by EntilZha — APPROVED** [review]

**Comment by rmaheshkumarblr** [comment]
No more changes. Looks good to me when I run it on my machine.

**Pull request merged**